airflow-core/pyproject.toml

```diff
@@ -141,7 +141,7 @@ dependencies = [
     # https://github.com/apache/airflow/issues/56369 , rework universal-pathlib usage
     "universal-pathlib>=0.2.6,<0.3.0",
     "uuid6>=2024.7.10",
-    "apache-airflow-task-sdk<1.2.0,==1.1.5",
+    "apache-airflow-task-sdk<1.2.0,>=1.1.5",
```

Comment:
@potiuk , Now that we have this protection, I'm returning this to >= instead of ==, any objections?

Reply:
We **still** do not test if the new task-sdk works with past Airflow versions. I would not really do it until we have back-compat tests maybe? 

Otherwise we might potentially get the situation that someone installs new task-sdk version without upgrading airflow (or - what's more probable - upgrades to newer version of airflow and rolls back, but task-sdk remains in newer version).